internal/config.go

```diff
@@ -14,49 +14,36 @@ var InstallDirectory = "/home/deck/.cryo_utilities"
 // LogFilePath Location of the log file
 var LogFilePath = filepath.Join(InstallDirectory, "cryoutilities.log")
 
-//////////////////////////
-// Recommended Settings //
-//////////////////////////
+type Tweak struct {
+	Location    string
+	Recommended string
+	Default     string
+}
+
+////////////////////
+// Tweak Settings //
+////////////////////
+
+var TweakList = map[string]Tweak{
+	"swappiness":               {Location: "/proc/sys/vm/swappiness", Recommended: "1", Default: "100"},
+	"page_lock_unfairness":     {Location: "/proc/sys/vm/page_lock_unfairness", Recommended: "1", Default: "5"},
+	"compaction_proactiveness": {Location: "/proc/sys/vm/compaction_proactiveness", Recommended: "0", Default: "20"},
+	"hugepages":                {Location: "/sys/kernel/mm/transparent_hugepage/enabled", Recommended: "always", Default: "madvise"},
+	"shmem_enabled":            {Location: "/sys/kernel/mm/transparent_hugepage/shmem_enabled", Recommended: "advise", Default: "never"},
+	"defrag":                   {Location: "/sys/kernel/mm/transparent_hugepage/khugepaged/defrag", Recommended: "0", Default: "1"},
```

Comment:
Naming is a bit inconsistent. defrag -> hugepages_defrag, shmem -> shem_enabled. Tweak list is actually a tweak table.

And I think its better to use named constants to reduce probability of a typing mistake.

Reply:
The names are on me, since I simply used the trailing end of the option to name them.

As for named constants, I could go either way. I think that this is much more extensible, so I lean very slightly to the way that @xeome is doing here.